modules/docs/mdx/changelog.stories.mdx

```diff
@@ -1,6 +1,6 @@
 import {Meta} from '@storybook/addon-docs/blocks';
-import Changelog from '../../CHANGELOG.md';
-import Markdown from '../../utils/storybook/Markdown.tsx';
+import Changelog from '../../../CHANGELOG.md';
```

Comment:
Does it make sense for this file to go here? It will be published with our docs package, but won't work in the Canvas site because the `CHANGELOG.md` file will not be published. So importing in the Canvas Site will result in a "module not found" error.

Maybe it has to be? Or it gets left where it was and becomes a Storybook-only MDX file - along with the "Getting Started" since that references the README.md file which also is not published.

Reply:
Hmm... That's a good point. We could leave it at the root and change it to `.mdx` - github can render them just fine, it will just skip any jsx tags. 

However, we actually may want to render the changelog on the Canvas Site. I left it at the root because it felt weird for a project to have that buried in a module folder. What do you think?

Reply:
It is a common convention to have a `CHANGELOG.md` file in the root of a project. If we want to render on the Canvas Site, we may have to copy the CHANGELOG.md file to the doc module directory as part of the build process and make sure the file is deployed to npm. I'm not sure if at that point this `MDX` wrapper is helpful to the Canvas Site since the Storybookisms have to be stripped away.

I think our Readme is going to be reduced to a much simpler version and have a link to more official documentation anyways. I'm not sure how useful that will be for the Canvas Site. I think having these MDX files are fine here even if not used by the Canvas Site to avoid too much odd configuration.